client/app/components/composites/MenuMobile/MenuMobile.css

```diff
@@ -60,7 +60,7 @@
   overflow-x: hidden;
   overflow-y: auto;
   will-change: transform;
-  transform: translateX(-288px);
+  transform: translateX(-300px);
```

Comment:
Good point, got caught already once when I only changed the other one and things didn't look right.